CHANGELOG.md

```diff
@@ -0,0 +1,44 @@
+# Changelog
+
+We strive to follow [SemVer](https://semver.org/) conventions. Per [item 4 in the spec](https://semver.org/#semantic-versioning-specification-semver), until the API stabilizes and we make an official `1.0.0` release, expect new features and/or breaking changes with every update to the minor version number. Patch numbers will continue to be backwards-compatible with a given minor version number.
+
+That said, this is research software, so expect some instability as we aim first and foremost to push the boundaries of what is possible.
+
+## 0.1.0
+
+Massive configuration API rewrite. This should make it easier to maintain and extend in the future.
+
+### New functions
+
+ - `enable_nan_injection!(n_inject = 1)`
+ - `disable_nan_injection!()`
+ - `enable_injection_recording!(recording_file::String = "ft_recording")`
+ - `set_injection_replay!(replay_file::String)`
+
+### Replaced functions
+
+ - `set_logger(kwargs...)` → `set_logger_config!(kwargs...)`
```

Comment:
> `!` for mutating functions

Should I take it out? I want to communicate that these functions are called solely for effect.

> short word for "configure"

What about `config_X`?

Reply:
Yes take `!` out. We should follow the Julia convention. If there's some other way to indicate side effects, use that.

`config_X` is ok. (It looks like a noun at first glance to me rather than a verb, but it is short!)